msd/SpeakerAward.py

```diff
@@ -0,0 +1,35 @@
+import json
+import azure.cognitiveservices.speech as speechsdk
+from dotenv import load_dotenv
+import os
+from pydub import AudioSegment
+
+class SpeakerAward():
+    
+    def __init__(self,diarizer):
+        self.diarizer = diarizer
+        self.json_outputs = None
+
+    def get_json(self):
+        load_dotenv()
+        self.json_outputs = []
+        i = 0
+        for segment, _, label in self.diarizer.diarization.itertracks(yield_label=True):
+            start = segment.start
+            end = start + segment.duration
+            speaker = label
+            self.json_outputs.append({'speaker':speaker,'start':start,'end':end})
```

Comment:
Comme t'auras gagner des lignes, hésite pas à créer ton dico à part 
```suggestion
            output = { 'speaker': speaker,'start': start,'end': end }
            self.json_outputs.append(output)
```
On gagne en lisibilité comme ça